CHANGELOG.md

```diff
@@ -2,13 +2,27 @@
 
 All notable changes to the DTS project will be documented in this file.
 
+## v1.2.13 - 2023-12-22
+
+* Added [dasharo_ectool](https://github.com/Dasharo/ec) for updating EC.
+* Added [bg-suite](https://github.com/9elements/converged-security-suite/tree/main/cmd/bg-suite).
+* Added [Dasharo configuration utility](https://github.com/Dasharo/dcu) that is
+  a tool designed to configure Dasharo firmware binary images. It includes task
+  such as customizing the boot logo, and setting unique UUIDs or Serial Numbers
+  in SMBIOS tables.
+* Added support fot exFAT and NTFS file systems.
+* Added ACPI BGRT support in kernel.
+* Added signature verification of Dasharo firmware binaries.
+* Added posibility to run updates from [local server](./README.md#testing-dasharo-firmware-updates-from-local-sources).
+* Fixed check in internet connection; issue [588](https://github.com/Dasharo/dasharo-issues/issues/588).
+
```

Comment:
@TomaszAIR we also check if charger is connected before update on laptops: https://github.com/Dasharo/meta-dts/commit/b137a37f0645c32408d1452bcc26b3210a0f944c

Reply:
And fixed that logo was not presered on some updates https://github.com/Dasharo/meta-dts/commit/6c6752bfee894a62fe63d2e4284ee32c261e4eaf

Reply:
@macpijan fixed https://github.com/Dasharo/meta-dts/pull/68/commits/9fcb43befbaed8fa53088532d8ea9c1af00e6499